### ISSUE TITLE
Vst.net compatibility problem with Cantabile?

### ISSUE THREAD

**Issue by ClaudioCas** [obiwanjacobi/vst.net#67]
Dear Jacobi,
thanks for your great library. I can't get vsts created with liberia to work with the Cantabile program. I tried to compile the examples in the library, Cantabile recognize the vst but crashes when trying to add it, The same dll with other host (Carla KsStudio) works correctly.

**To Reproduce**
Compile 64bit of a sample Vst of the library
Try to add it to Cantabile Project

**Desktop (please complete the following information):**
 - OS: Win 7/10
 - Cantabile 4

**Comment by obiwanjacobi** [comment]
No, not really. Such a memory dump is only useful if you now the Cantabile source code.
Do you know of any other plugins that make Cantabile crash -or- comparable plugins that work okay?

**Comment by ClaudioCas** [comment]
Thanks for the clarification on the mdmp.
No, I haven't encountered any other crashes with other vsts.
I also opened a thread on the forum, but I haven't received any useful information.
If you have the possibility to contact the developer directly https://www.cantabilesoftware.com/contact maybe you can find the problem and a solution to avoid the crash.
Thank you for your kindness and for the time you have dedicated to me

**Comment by obiwanjacobi** [comment]
I don't have the time to research this.

You could try to attach a debugger (Visual Studio) to the Cantabile exe and then add the plugin and make it crash. Perhaps you'll see what (VST.NET or Cantabile) call fails...
If you plan to further investigate, please share your findings here.

Sorry, I cannot help you further...

**Comment by ClaudioCas** [comment]
Dear Jacobi,
I understand, most likely the problem is with the software and not with the library since the vst are loaded from other hosts without problems.
Thanks for the time you gave me.
